docs/tutorials/my-first-custom-action.rst

```diff
@@ -0,0 +1,255 @@
+Your first custom action!
+######################################################
+
+Learn how to automate everything with Robusta, in *15 minutes* or less :)
+
+Goals
+---------------------------------------
+We’re gonna learn how to create a custom playbook using a short and made-up (but realistic) scenario.
+
+First, we’ll describe the the scenario, and then we’ll implement a Robusta automation (Playbook) that detect this issue immediately and recommends a fix!
+
+You’re more than welcome to follow along the scenario on your own machine 💻
+
+.. note::
+
+    It is recommended to read :ref:`Example Playbook` before starting this guide.
+
+The scenario
+---------------------------------------
+You want to create a new pod with the image “ngnix”!
+
+Being the smart person that you are, you decide save some time by copying and pasting an existing YAML file you have in your computer, and change the pod name and image to ngnix.
+
+.. code-block:: yaml
+
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: nginx
+      labels:
+        env: test
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+        imagePullPolicy: IfNotPresent
+      nodeSelector:
+        spoiler: alert
+
+Now you start the pod by running the following command:
+
+.. code-block:: bash
+
+    $ kubectl apply -f nginx-pod.yaml
+    pod/nginx created
+
+but for some reason it it not starting (note it’s “Pending” Status):
+
+.. code-block:: bash
+
+    $ kubectl get pods
+
+    NAME                                                     READY   STATUS    RESTARTS   AGE
+
+    nginx                                                    0/1     Pending   0          5h19m
+
+You wait a few minutes, but is remains the same.
```

Comment:
Do you think we should mention the Robusta UI here. 
Like, if you're using the Robusta UI you can see it there as well

Reply:
not sure I understand . What do you see in Robusta UI? You won't see the pod (like Lens for example)

Reply:
Won't you see the warning event on the pod in the timeline?

Reply:
done